IdentityCore/src/broker_operation/request/token_request/MSIDBrokerOperationBrowserTokenRequest.h

```diff
@@ -36,12 +36,14 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) MSIDAADAuthority *authority;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->